src/graphics/program-lib/chunks/standard/frag/normalDetailMap.js

```diff
@@ -7,17 +7,17 @@ vec3 blendNormals(vec3 n1, vec3 n2) {
     // https://blog.selfshadow.com/publications/blending-in-detail/#detail-oriented
     n1 += vec3(0, 0, 1);
     n2 *= vec3(-1, -1, 1);
-    return normalize(n1*dot(n1, n2)/n1.z - n2);
+    return n1 * dot(n1, n2) - n2 * n1.z;
```

Comment:
why has this line changed?

Reply:
but how about `/n1.z` is now `*n1.z'?

Reply:
I had changed the formula to the one specified by the linked article. I just did another a quick test though and actually the original is better. I've reverted that now, thanks!